### PR TITLE
[docs] [integrations] Apache Flink, Kinesis data stream, Hevo Data

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/integrations/apache-flink/

### CODE REVIEW

**Review by ddhodge — APPROVED** [review]
LGTM

**Review by ashetkar — APPROVED** [review]

File: docs/content/preview/integrations/_index.md [hunk 1]
```diff
@@ -113,6 +120,13 @@ For information on YugabyteDB-compatible drivers and ORMs, refer to [Drivers and
     </a>
   </li>
 
+  <li>
+    <a href="kinesis/">
```
Comment (ddhodge) [inline docs/content/preview/integrations/_index.md:124]: broken link. maybe rename the file.

File: docs/content/preview/integrations/_index.md [hunk 2]
```diff
@@ -113,6 +120,13 @@ For information on YugabyteDB-compatible drivers and ORMs, refer to [Drivers and
...
+      <img src="/images/section_icons/develop/ecosystem/kinesis.png">
+      Kinesis Data Stream
```
Comment (ddhodge) [inline docs/content/preview/integrations/_index.md:126]:
```suggestion
      Kinesis Data Streams
```

File: docs/content/preview/integrations/kineses.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+- Create a Kinesis Data Stream from AWS Management Console. Note the name of the stream you create.
+- Start a YugabyteDB cluster. Refer [YugabyteDB Prerequisites](../../tools/#yugabytedb-prerequisites).
+- Create a CDC stream using following command:
+
+    ```sh
+    ./bin/yb-admin --master_addresses 127.0.0.1:7100 create_change_data_stream ysql.yugabyte
+    ```
+
+- Create a table using [YSQLSH](../../admin/ysqlsh/#starting-ysqlsh) as follows:
```
Comment (ddhodge) [inline docs/content/preview/integrations/kineses.md:30]:
```suggestion
- Create a table using [ysqlsh](../../admin/ysqlsh/#starting-ysqlsh) as follows:
```

File: docs/content/preview/integrations/kineses.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    ```sql
+    CREATE TABLE users(
+    id         bigserial PRIMARY KEY, created_at timestamp,
+    name       text, email      text, address    text,
+    city       text, state      text, zip        text,
+    birth_date text, latitude   float, longitude  float,
+    password   text, source     text);
+    ```
+
+- Write a Java application which will use the Debezium connector to receive CDC data from YugabyteDB and write to Kinesis Data Stream.
```
Comment (ddhodge) [inline docs/content/preview/integrations/kineses.md:41]:
```suggestion
- Write a Java application which will use the Debezium connector to receive CDC data from YugabyteDB and write to Kinesis Data Streams.
```

File: docs/content/preview/integrations/kineses.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+Code snippets of a sample code is as follows:
```
Comment (ddhodge) [inline docs/content/preview/integrations/kineses.md:42]:
```suggestion
The following code snippet shows an example implementation:
```

File: docs/content/preview/integrations/kineses.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+       PutRecordResult putRecordResult = kinesisClient.putRecord( putRecordRequest );
+       lastSequenceNumber = putRecordResult.getSequenceNumber();
+
+    // Now run the engine asynchronously
+    ExecutorService executor = Executors.newSingleThreadExecutor();
+    executor.execute(engine);
+    ```
+
+- As you insert records into the users table, you can check the records arriving in the Kinesis Data Stream.
+For some INSERT DMLs, refer to `users.sql` script in [CDC examples](https://github.com/yugabyte/cdc-examples/blob/main/cdc-quickstart-kafka-connect/scripts/users.sql).
```
Comment (ddhodge) [inline docs/content/preview/integrations/kineses.md:124]:
```suggestion
For some INSERT DMLs, refer to the `users.sql` script in [CDC examples](https://github.com/yugabyte/cdc-examples/blob/main/cdc-quickstart-kafka-connect/scripts/users.sql).
```

File: docs/content/preview/integrations/kineses.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+       putRecordRequest.setSequenceNumberForOrdering( lastSequenceNumber );
+       PutRecordResult putRecordResult = kinesisClient.putRecord( putRecordRequest );
+       lastSequenceNumber = putRecordResult.getSequenceNumber();
+
+    // Now run the engine asynchronously
+    ExecutorService executor = Executors.newSingleThreadExecutor();
+    executor.execute(engine);
+    ```
+
+- As you insert records into the users table, you can check the records arriving in the Kinesis Data Stream.
```
Comment (ddhodge) [inline docs/content/preview/integrations/kineses.md:123]:
```suggestion
- As you insert records into the users table, you can check the records arriving in the Kinesis data stream.
```

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/integrations/kineses.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
+---
```
Comment (ashetkar) [inline docs/content/preview/integrations/kineses.md:1]: nit: name of the file should be `kinesis.md`

File: docs/content/preview/integrations/kineses.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+[Amazon Kinesis](https://aws.amazon.com/kinesis/) Data Streams enables applications to collect and process large streams of data records in real time. These applications can send the processed records to dashboards, use them to generate alerts, dynamically change pricing and advertising strategies, or send data to a variety of other AWS services.
+
+YugabyteDB has a [Debezium connector](https://github.com/yugabyte/debezium-connector-yugabytedb) which you can use to read changes to a table and then write those into Kinesis Data Streams using the AWS SDK for Java and the Kinesis Producer Library.
+
+## Connect
+
+To connect your YugabyteDB database to Kinesis:
+
+- Create a Kinesis Data Stream from AWS Management Console. Note the name of the stream you create.
```
Comment (ashetkar) [inline docs/content/preview/integrations/kineses.md:22]: Create a Kinesis Data Stream from ... ->

Create a [Kinesis Data Stream](https://docs.aws.amazon.com/streams/latest/dev/tutorial-stock-data-kplkcl-create-stream.html) from ...

**Review by ashetkar — APPROVED** [review]

File: docs/content/preview/integrations/hevodata.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+Hevo ingests your data from [Sources](https://docs.hevodata.com/sources/), applies transformations on it, and brings it to a [Schema Mapper](https://docs.hevodata.com/pipelines/schema-mapper/) using which you can define how your data must be stored at the [Destination](https://docs.hevodata.com/destinations/).
+
+Hevo supports PostgreSQL among many other databases as a source. It can also work with YugabyteDB as a source using PostgreSQL's configuration with some changes.
+
+## Connect
+
+To add YugabyteDB as a source in [Pipeline](https://docs.hevodata.com/pipelines/), do the following:
+
+1. Start a YugabyteDB cluster. Refer [YugabyteDB Prerequisites](../../tools/#yugabytedb-prerequisites).
+1. Create a new Pipeline by selecting PostgreSQL as the source type.
```
Comment (ddhodge) [inline docs/content/preview/integrations/hevodata.md:27]:
```suggestion
1. In the Hevo UI, create a new Pipeline by selecting PostgreSQL as the source type.
```

File: docs/content/preview/integrations/hevodata.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+1. Provide an appropriate hostname and port (Default for YugabyteDB is 5433) for your YugabyteDB cluster as per the following illustration:
+
+   ![pipeline](/images/section_icons/develop/ecosystem/hevodata-setup.png)
```
Comment (ddhodge) [inline docs/content/preview/integrations/hevodata.md:30]: do we have a better picture? day theme? not so low res?

File: docs/content/preview/integrations/hevodata.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Select "Table" as the "Ingestion Mode" for YugabyteDB. Custom SQL can also be opted in which case, you also need to provide the actual SQL to fetch data from your tables. "Logical Replication" is not supported.
```
Comment (ddhodge) [inline docs/content/preview/integrations/hevodata.md:32]:
```suggestion
1. Under **Select an Ingestion Mode**, choose **Table**. You can also choose **Custom SQL**, in which case you also need to provide the actual SQL to fetch data from your tables. **Logical Replication** is not supported.
```

File: docs/content/preview/integrations/hevodata.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+1. Provide the database and schema name where the table data is present and other connection details.
+1. Select the tables which you want to ingest into Hevo.
+1. Select a value for the "Query Mode". If your table has an incrementing column and a timestamp column which gets updated for updates, you can choose "Change Data Capture". The mode "xmin" is not supported in YugabyteDB.
```
Comment (ddhodge) [inline docs/content/preview/integrations/hevodata.md:35]:
```suggestion
1. Select a value for the **Query Mode**. If your table has an incrementing column and a timestamp column which gets updated for updates, you can choose **Change Data Capture**. **xmin** is not supported in YugabyteDB.
```

File: docs/content/preview/integrations/hevodata.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+1. Select and configure a destination as per your requirement.
+
+After the pipeline is created, you can see that the data from your YugabyteDB cluster gets ingested into the destination via this Hevo pipeline.
```
Comment (ddhodge) [inline docs/content/preview/integrations/hevodata.md:38]:
```suggestion
After the pipeline is created, data from your YugabyteDB cluster gets ingested into the destination via the Hevo pipeline.

```

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/integrations/hevodata.md [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+[Hevo Data](https://hevodata.com/) is a no-code data movement platform which lets you set up your database or data warehouse for analytics and is designed to process billions of records.
+After all the historical data in the source is ingested, any new and changed data thereafter is ingested as incremental data. In case of RDBMS sources like PostgreSQL, you can define how Hevo fetches this data, using ingestion modes, and query modes.
+
+Hevo ingests your data from [Sources](https://docs.hevodata.com/sources/), applies transformations on it, and brings it to a [Schema Mapper](https://docs.hevodata.com/pipelines/schema-mapper/) using which you can define how your data must be stored at the [Destination](https://docs.hevodata.com/destinations/).
+
+Hevo supports PostgreSQL among many other databases as a source. It can also work with YugabyteDB as a source using PostgreSQL's configuration with some changes.
+
+## Connect
+
+To add YugabyteDB as a source in [Pipeline](https://docs.hevodata.com/pipelines/), do the following:
```
Comment (ddhodge) [inline docs/content/preview/integrations/hevodata.md:24]:
```suggestion
To add YugabyteDB as a source in Hevo, create a [Pipeline](https://docs.hevodata.com/pipelines/) as follows:
```

File: docs/content/preview/integrations/hevodata.md (hunk 11 above)
Comment (aishwarya24) [reply docs/content/preview/integrations/hevodata.md:30]: cc @ashetkar

**Pull request merged**